.github/workflows/release-candidate.yaml

```diff
@@ -15,121 +15,90 @@ env:
           repo: docs_new
-          github_token: ${{ secrets.GH_PAT }}
-          workflow_file_name: test-release.yml
+          github_token: ${{ secrets.RAFAL_ACCESS_TOKEN }}
+          workflow_file_name: parser-test.yaml
           ref: master
           client_payload: '{"husarion_ugv_branch": "ros2-devel"}'
 
-  # TODO: Add unit testing for panther_ros when ready
-  unit_test_panther_ros:
-    name: Run unit tests for panther_ros
+  # TODO: Add unit testing when ready
+  unit_tests:
+    name: Unit tests
     runs-on: ubuntu-22.04
     steps:
-      - name: Trigger repository build workflow
-        run: echo "Unit tests for panther_ros are not fully implemented yet -> SKIPPING!"
+      - name: Run unit tests
+        run: echo "Unit tests are not fully implemented yet -> SKIPPING!"
 
-  create_release_candidate_branches:
-    name: Create release candidate branches
+  docker:
+    name: Docker
     runs-on: ubuntu-22.04
-    needs:
-      - unit_test_panther_ros
-    strategy:
-      matrix:
-        repo: [panther_ros, panther-docker, panther-rpi-os-img]
     steps:
-      - name: Create test branch
-        uses: GuillaumeFalourd/create-other-repo-branch-action@v1.5
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Update docker image tag
+        uses: mikefarah/yq@v4.43.1
         with:
-          repository_owner: husarion
-          repository_name: ${{ matrix.repo }}
-          new_branch_name: ${{ env.RC_BRANCH_NAME }}
-          new_branch_ref: ros2-devel
-          access_token: ${{ secrets.GH_PAT}}
+          cmd: |
+            sed -i 's/\(husarion\/panther:humble-\)[^[:space:]]*/\1${{ env.RC_BRANCH_NAME }}/g' docker/demo/compose.hardware.yaml
+            sed -i 's/\(husarion\/panther:humble-\)[^[:space:]]*/\1${{ env.RC_BRANCH_NAME }}/g' docker/demo/compose.rviz.yaml
+            sed -i 's/\(husarion\/panther-gazebo:humble-\)[^[:space:]]*/\1${{ env.RC_BRANCH_NAME }}/g' docker/demo/compose.simulation.yaml
 
-  update_tags_in_compose:
-    name: Update Docker image tags in compose
-    runs-on: ubuntu-22.04
-    needs:
-      - create_release_candidate_branches
-    steps:
-      - name: Trigger repository build workflow
-        uses: convictional/trigger-workflow-and-wait@v1.6.1
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v9
         with:
-          owner: husarion
-          repo: panther-docker
-          github_token: ${{ secrets.GH_PAT }}
-          workflow_file_name: update-tags-in-compose.yaml
-          ref: ${{ env.RC_BRANCH_NAME }}
-          client_payload: |
-            {
-              "branch_name": "${{ env.RC_BRANCH_NAME }}",
-              "image_tag": "humble-${{ env.RC_BRANCH_NAME }}"
-            }
+          message: Update tags in compose files
+          author_name: action-bot
+          author_email: action-bot@action-bot.com
+          new_branch: ${{ env.RC_BRANCH_NAME }}
 
-  build_and_push_docker_images:
-    name: Build panther docker images
```

Comment:
You removed this step that is required

Reply:
This step is right now just executing `./.github/workflows/build-docker.yaml`